ssv/spectest/tests/runner/full_happy_flow_sc.go

```diff
@@ -2,24 +2,25 @@ package runner
 
 import (
 	"github.com/attestantio/go-eth2-client/spec"
+
 	"github.com/bloxapp/ssv-spec/qbft"
 	qbftcomparable "github.com/bloxapp/ssv-spec/qbft/spectest/comparable"
 	"github.com/bloxapp/ssv-spec/ssv"
 	ssvcomparable "github.com/bloxapp/ssv-spec/ssv/spectest/comparable"
 	"github.com/bloxapp/ssv-spec/types"
 	"github.com/bloxapp/ssv-spec/types/testingutils"
-	comparable2 "github.com/bloxapp/ssv-spec/types/testingutils/comparable"
+	typescomparable "github.com/bloxapp/ssv-spec/types/testingutils/comparable"
 )
 
-// fullHappyFlowStateComparison returns state comparison object for the FullHappyFlow spec test
-func fullHappyFlowStateComparison() *ssvcomparable.StateComparison {
+// fullHappyFlowSyncCommitteeContributionSC returns state comparison object for the FullHappyFlow SyncCommitteeContribution spec test
+func fullHappyFlowSyncCommitteeContributionSC() *qbftcomparable.StateComparison {
```

Comment:
@GalRogozinski we changed ssvcomparable.StateComparison to qbftcomparable.StateComparison
If its the same, does it make sense to move it to one place?